### PR TITLE
14 Filling form - fixed test site URL and files ext

### PR DESCRIPTION
**Pull request by JorgeVQA**
Done this part of the workshop, please check and give me your thoughts.

_**FYI - There are couple of fields not required for this test:**_
- dateOfBirth (it is already filled) - not required
- Subjects - not required

### CODE REVIEW

File: cypress/page/personal-input.page.ts [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+    cy.contains(this.stateField)
+      .click({ force: true })
+      .type(`${baseInfo.state}{enter}`);
+
+    cy.contains(this.cityField)
+      .click({ force: true })
+      .type(`${baseInfo.city}{enter}`);
+
+    cy.get(this.submitFormButton).click({ force: true });
```
Comment (manuelq12) [inline cypress/page/personal-input.page.ts:85]: I don't think you need force:true here, you could have also used cy.submit()

File: cypress/support/commands.ts [hunk 2]
```diff
@@ -23,3 +23,9 @@
 //
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+// Ignoring uncaught exceptions since errors from external apps should not stop de workshop
+Cypress.on("uncaught:exception", (err, runnable) => {
+  // returning false here prevents Cypress from
+  // failing the test
+  return false;
```
Comment (manuelq12) [inline cypress/support/commands.ts:30]: If you have this yo don't need the statement on the before

File: cypress/page/personal-input.page.ts [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+    cy.visit(this.inputPageUrl);
+  }
+
+  // 2.  Fill hobbiesList array with all available hobbies
+  private enterHobbies(hobbiesList: string[]): void {
+    for (let hobbie in hobbiesList) {
+      cy.get(this.hobbiesField)
+        .contains("label", hobbiesList[hobbie])
+        .siblings()
+        .click({ force: true });
```
Comment (manuelq12) [inline cypress/page/personal-input.page.ts:47]: You could have used cy.check()

**Review by manuelq12 — APPROVED** [review]
I don't think this requires changes, but I gave you some input of the code

File: cypress/page/personal-input.page.ts (hunk 1 above)
Comment (JorgeVQA) [reply cypress/page/personal-input.page.ts:85]: I will fix it for the next PR.   Thanks for the advise.

File: cypress/support/commands.ts (hunk 2 above)
Comment (JorgeVQA) [reply cypress/support/commands.ts:30]: I will fix it for the next PR.   Thanks for the advise.

File: cypress/page/personal-input.page.ts (hunk 3 above)
Comment (JorgeVQA) [reply cypress/page/personal-input.page.ts:47]: I will fix it for the next PR.   Thanks for the advise.

**Pull request merged**